### PR TITLE
Sparkbox Sprint 8

### PR DESCRIPTION
**Pull request by kaseybon**
# READY FOR REVIEW

## Summary
- WYSIWYG Styles [STN-127](https://sparkbox.atlassian.net/browse/STN-127) | PR #494 
- Local Footer Styles and Implementation [STN-172](https://sparkbox.atlassian.net/browse/STN-172) | PR #496 
- QA Feedback from last sprint [STN-184](https://sparkbox.atlassian.net/browse/STN-184) | PR #497 
- Hero option 1 styles [STN-125](https://sparkbox.atlassian.net/browse/STN-125) | PR #495 

✅ Passing CodeClimate: https://codeclimate.com/github/SU-HSDO/suhumsci/pull/498

## Need Review By (Date)
asap (2/25)

## Urgency
high

## Steps to Test
- ⚠️ TODO: We've verified that this branch passes the code climate and circle CI tests against the develop branch.

1. See other PRs "steps to test" if you'd like to test locally for those changes.
2. Verify code looking at the file changes here.
3. Go into `humsci_basic` folder directory and run `npm i` and `npm run test` to verify tests pass.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)

### CODE REVIEW

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr498-x5eyppcnpidw1z9tdfpnbjvg9wjmt5q0.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5e548245008a56cb813fc647

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr498-x5eyppcnpidw1z9tdfpnbjvg9wjmt5q0.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5e548245008a56cb813fc647

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr498-x5eyppcnpidw1z9tdfpnbjvg9wjmt5q0.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5e548245008a56cb813fc647

File: docroot/themes/humsci/humsci_colorful/theme-settings.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+/**
+ * @file
+ * Provides an additional config form for theme settings.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Url;
+use Drupal\Core\Link;
+use Drupal\Core\Render\Markup;
```
Comment (pookmish) [inline docroot/themes/humsci/humsci_colorful/theme-settings.php]: unsused `use` statements

**Review by pookmish — CHANGES_REQUESTED** [review]
I've also noticed the behat tests are failing. I'll open a PR and request that be fixed as part of this

File: docroot/themes/humsci/humsci_colorful/theme-settings.php (hunk 1 above)
Comment (kaseybon) [reply docroot/themes/humsci/humsci_colorful/theme-settings.php]: @pookmish I pushed a fix removing those statements.

**Comment by pookmish** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr498-x5eyppcnpidw1z9tdfpnbjvg9wjmt5q0.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5e548245008a56cb813fc647

**Pull request merged**